sparse/_coo/common.py

```diff
@@ -125,7 +125,7 @@ def tensordot(a, b, axes=2):
     ndb = b.ndim
     equal = True
     if nda == 0 or ndb == 0:
-        pos = 0 if nda == 0 else 1
+        pos = int(nda == 0)
```

Comment:
```suggestion
        pos = int(nda != 0)
```

Sorry, my bad on this one.